kombu/entity.py

```diff
@@ -98,7 +98,7 @@ class Exchange(MaybeChannelBound):
 
 
                 .. _`AMQP in 10 minutes: Part 4`:
-                    http://bit.ly/amqp-exchange-types
+                    http://bit.ly/2rcICv5
```

Comment:
Yep. Would it be fine to switch to HTTPS in the same commit for the other bit.ly link (Lamport's distributed mutex)?